command/token_create.go

```diff
@@ -82,8 +82,9 @@ Usage: vault token-create [options]
   This token will be created as a child of your token. The created token
   will inherit your policies, or can be assigned a subset of your policies.
 
-  A lease can also be associated with the token. If a lease is associated,
-  it will expire after that amount of time unless it is renewed.
+  A lease can also be associated with the token. If a lease is not associated
+  with the token, then it cannot be renewed. And if a lease is associated with
```

Comment:
Don't start a sentence with "and". It's okay in this case to put "If a lease" twice, so just remove the and and capitalize the if  :-)